configure

```diff
@@ -696,7 +696,7 @@ esac
 
 case "$target" in
   # PR#5088: autodetection is unreliable on ARM.  PR#5280: also on MIPS.
-  sparc*-*-*|hppa*-*-*|arm*-*-*|mips*-*-*)
```

Comment:
Don't remove, SPARC is still supported in bytecode, and this alignment business affects the bytecode system.